.travis.yml

```diff
@@ -6,4 +7,4 @@ before_install:
 
 script:
     - python3 -m unittest discover -s metatests
-    - docker run autograder /bin/sh -c "chmod +x source/metatests/integration.sh && ./source/metatests/integration.sh"
```

Comment:
Ok, this change was supposed to be present in the original pull request.  I'm glad it's here now.